Cargo.toml

```diff
@@ -35,7 +35,7 @@ colored = "2.1"
 serde = "1.0"
 rust-embed = "8.2.0"
 is-terminal = "0.4.10"
-wasmprof = "0.3.0"
+wasmprof = { path = "../wasmprof" }
```

Comment:
Will change this to `0.4.0` once [the wasmprof PR](https://github.com/Shopify/wasmprof/pull/40) ships